documentation.md

```diff
@@ -19,11 +19,11 @@ For previous versions of the specification, please see the [Specification Links]
 Meta-schemas
 ------------
 
-The meta-schemas are the schemas which define the JSON Schema and Hyper-Schema formats.
-
+The meta-schemas define the JSON Schema and Hyper-Schema formats using the JSON Schema format.
```

Comment:
I think this might flow a bit better, see if you agree:

> The meta-schemas are schemas against which other schemas are validated.  They are self-descriptive: the JSON Schema meta-schema validates itself, while the JSON Hyper-Schema meta-schema both validates itself and defines its own "self" link.

Alternatively, we could keep the original wording, except replace "define" with "describe", and once this is merged we can tinker with the wording further.  But this really isn't the point of this PR so I don't want to hold it up if this is the only discussion.

BTW "meta-schema" is a generic noun, so it is not capitalized.